.github/workflows/test.yaml

```diff
@@ -15,8 +15,20 @@ jobs:
     steps:
 
     - uses: actions/checkout@v4
-      # TODO: Check out the submodules as well and then launch a test graph with docker compose to run the
-      # integration tests
+      with:
+        submodules: recursive
+
+    - name: Launch test graph
+      run: docker compose up -d test_graph
+
+    # TODO: Check if this is necessary, since the Python dependencies setup also takes some time
+    - name: Sleep to allow graph setup
+      uses: jakejarvis/wait-action@master
+      with:
+        time: '5s'
+
+    - name: Check running containers
+      run: docker ps
```

Comment:
Is this so we can check whether containers are running in the workflow logs?

Reply:
Yes, just as a sanity check :)